Compile warning

<!-- (At least include the following, feel free to add if you have more content) -->
Compile warning:
```
starrocks/be/src/io/random_access_file.cpp: In member function ‘virtual starrocks::Status starrocks::io::RandomAccessFile::read_at_fully(int64_t, void*, int64_t)’:
/home/decster/projects/starrocks/be/src/io/random_access_file.cpp:10:63: warning: pointer of type ‘void *’ used in arithmetic [-Wpointer-arith]
   10 |         ASSIGN_OR_RETURN(auto n, read_at(offset + nread, data + nread, count - nread));

```

--- a/be/src/io/readable.cpp
+++ b/be/src/io/readable.cpp
@@ -7,7 +7,7 @@ namespace starrocks::io {
 Status Readable::read_fully(void* data, int64_t count) {
     int64_t nread = 0;
     while (nread < count) {
-        ASSIGN_OR_RETURN(auto n, read(data + nread, count - nread));
+        ASSIGN_OR_RETURN(auto n, read(static_cast<uint8_t*>(data) + nread, count - nread));
         nread += n;
         if (n == 0) {
             return Status::IOError("cannot read fully");

